src/openrct2/scripting/bindings/ride/ScTrackSegment.cpp

```diff
@@ -194,42 +194,46 @@ std::vector<DukValue> ScTrackSegment::getSubpositions(uint8_t trackSubposition,
     return result;
 }
 
+static DukValue _trackCurveToString(duk_context* ctx, TrackCurve curve)
```

Comment:
We have EnumMap for this purpose, right?

Reply:
That one works the other way.

Reply:
EnumMap is declared both ways. Note:
```cpp

    std::string_view operator[](T k) const
    {
        auto it = find(k);
        return it->first;
    }

    T operator[](std::string_view k) const
    {
        auto it = find(k);
        return it->second;
    }
```